app/models/profile_link_site.rb

```diff
@@ -11,5 +11,5 @@
 # rubocop:enable Metrics/LineLength
```

Comment:
Could you reannotate here?  You should have the addition of a `validate_find` and `validate_replace` column across here, the spec, and factory